lca8s8-d1e776/cod-dlc9qq_lca8s8-d1e776.xml

```diff
@@ -0,0 +1,359 @@
+  </teiHeader>
+  <text xml:lang="la">
+    <front>
+      <div xml:id="starts-on">
+        <pb ed="#P" n="234-v"/>
+      </div>
+    </front>
+    <body>
+      <div xml:id="lca8s8-d1e776">
+        <head>Caput 56 [Bnf 12926 Transcription]</head>
+        <p>
+          <lb ed="#P"/>Caput 56m
+        </p>
+        <p>
+          <lb ed="#P"/>¶ Tertio docet <name>Ioannes Maioris</name> populum 
+          <lb ed="#P"/>qui christi fidem suscepit: debere si eam
+          <lb ed="#P"/>suscepit ex animo velle ut princeps suus 
+          <lb ed="#P"/>qui in paganismo perseuerat deponatur.
+          <lb ed="#P"/>Equidem populus natura vel quasi na<lb ed="#P" break="no"/>tura 
+          diligit antiquum principem suum
+          <lb ed="#P"/>secundum <ref>Philosophum 8o Ethicorum, c. 7.</ref> et <ref>3 po<lb ed="#P" break="no"/>liticorum
+          c. 10.</ref> <ref>Augustinum de ciuitate Dei 
+          <lb ed="#P"/>libro 19, c. 16.</ref> Ideo populus non minus 
+          <lb ed="#P"/>difficulter hoc admittat quam nouam 
+          <lb ed="#P"/>religionem. Presertim si pro deposito ve<lb ed="#P" break="no"/>tere 
+          domino: substituatur exterus, quod 
+          populo ingratum et odiosum esse solet. 
+          <lb ed="#P"/>Ideo deus omnipotens <ref target="http://scta.info/resource/deut17_14 http://scta.info/resource/deut17_15">Deu. 17</ref> praeci<lb ed="#P"/>pit
+          Iudaeis ut si regem electuri essent
+          <pb ed="#P" n="235-r"/>
+          <lb ed="#P"/>eligerent ex sua ipsorum. Cuius rei rat<lb ed="#P"/>ionem 
+          assignat <ref>sanctus Thomas 1 2 226, q. 105<lb ed="#P"/>a
+          <lb ed="#P"/>a. 1 ad 2m.</ref> Ideo non est sinistre sus<lb ed="#P"/>picandum
+          de populo nouiter conuerso: ex 
+          <lb ed="#P"/>eo quod nolit veterem principem suum de<lb ed="#P"/>poni:
+          etiam si paganus sit. Spectare 
+          <lb ed="#P"/>enim potest illum aliquando susceptu<lb ed="#P"/>rum 
+          fidem. Deinde populus recenter 
+          <lb ed="#P"/>conuersus non est mirandus quod non ha<lb ed="#P"/>beat 
+          tantum Euangelici vigoris: ut stat<lb ed="#P"/>im
+		  exutis omnibus affectibus exosum
+		  <lb ed="#P"/>habeat veterem principem suum mult<lb ed="#P"/>orum
+          annorum consuetudine cogni<lb ed="#P"/>tum. 
+          Ideo ipsorum affectibus etiam si 
+          <lb ed="#P"/>non sint pure christiani aliquid permit<lb ed="#P"/>tendum
+          et dandum est, autore <ref>Gregorio
+          <lb ed="#P"/>libro 12 epistola 31.</ref> et <ref>Ad interrogationes 
+          <lb ed="#P"/>Augustini c. 7</ref> quarum fragmenta 
+          <lb ed="#P"/>leguntur <ref>35 q. 2 et 3, c. quaedam 
+          <lb ed="#P"/>lex</ref>. <ref>Ne bonum (inquit) quod infirma ad
+          <lb ed="#P"/>huc radice plantatum est eruatur: sed 
+          <lb ed="#P"/>coeptum firmetur et usque ad perfectio<lb ed="#P"/>nem
+          custodiatur.<ref> Haec ille. Ex consequen<lb ed="#P"/>ti 
```

Comment:
```suggestion
          <lb ed="#P"/>lex</ref>. <quote>Ne bonum (inquit) quod infirma ad
          <lb ed="#P"/>huc radice plantatum est eruatur: sed 
          <lb ed="#P"/>coeptum firmetur et usque ad perfectio<lb ed="#P"/>nem
          custodiatur.</quote> Haec ille. Ex consequen<lb ed="#P"/>ti 
```

Reply:
Notice how your closing </ref> was missing the /.

Reply:
Got it. I'll try to run my own validation before committing next time.